chapters/12-project-management.Rmd

```diff
@@ -9,6 +9,35 @@
     - If you prefer GitHub Projects, create a GitHub Project for your package. Organize your issues from left to right in some version of "to-do", "in-progress", and "complete".
 - Milestones
 
+## Learning objectives
+
+* Use GitHub issues, issue labels, and roadmaps to coordinate project tasks and goals
+* Create GitHub Projects and GitHub Milestones for long-term project management
+* Explain the purpose of Projects and Milestones and when to use each
+
+## Outline
+
+<!-- We have a choice about the order of presenting this material: should we start at the top, with high-level project management, and then go over the components (i.e. GitHub projects / milestones -> roadmap -> issues) or should we start small with issues and build up to their combined uses for project management? Issues as bug reports and feature requests from the community are now covered in chapter 10; here we get into issue categorization, labeling, and milestones and project boards.-->
+
+- Project management with GitHub Issues
+  - Text: Beyond bugs: how GitHub issues can help you keep track of tasks and communicate with your community ([reference](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#project-structure-and-organization)). How to label issues and link issues to other parts of your project.
+  - Exercise: create at least three issue labels for the `zipfs` repository. Label the existing issues so that each has at least one label.
```

Comment:
There's a function that can help create some good defaults with `usethis::use_tidy_labels()`. You can also create your own labels with `usethis::use_github_labels()` if you want to do it through R. Not sure how much we want to get them to do this over manually creating them.

Reply:
Oooh interesting. I like the tidy labels, they seem good - might be good to start with them and then have learners remove or rename them from there?